app/demo-interactor/KNDemoIO.hh

```diff
@@ -34,8 +37,8 @@ struct KNDemoResult
     using size_type = celeritas::size_type;
 
     std::vector<double>    time;           //!< Real time per step
-    std::vector<double>    edep;           //!< Energy deposition per step
-    std::vector<size_type> alive;          //!< Num living tracks
+    std::vector<size_type> alive;          //!< Num living tracks per step
+    std::vector<double>    edep; //!< Energy deposition along the grid
```

Comment:
```suggestion
    std::vector<size_type> alive;          //!< Num living tracks per step
    std::vector<double>    edep;          //!< Energy deposition along the grid
```

Reply:
Thanks! Must've been left unformatted from the previous merge. I reformatted all the stuff in this directory.